connector/google/google.go

```diff
@@ -0,0 +1,192 @@
+// Package google implements logging in through OpenID Connect providers.
+package google
+import (
+	"golang.org/x/oauth2"
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+	"github.com/coreos/go-oidc"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/oauth2/google"
+	"github.com/coreos/dex/connector"
+	directory "google.golang.org/api/admin/directory/v1"
```

Comment:
let's not use an alias for this import. it matches better with the other code in the repo to not do it